jupyter_book/commands/__init__.py

```diff
@@ -486,7 +486,7 @@ def sphinx(ctx, path_source, config, toc):
     )
 
     sphinx_config, config_meta = get_final_config(
-        Path(toc) if toc else Path(),
+        Path(toc) if toc else Path(path_source + "/_toc.yml"),
```

Comment:
hmmm - it looks like there are two variables called `path_source` and `path_src` - can you rename one of them to help disambiguate?

Reply:
also just double checking that the `/` will work on windows as expected? If not you could always do `.joinpath('toc.yml')`

Reply:
Thanks @choldgraf , used `joinpath`. Old habit sometimes die hard 😬. 

Have renamed `path_src` to `full_path_source`.
The difference between the two variables, when running in source dir :

1) path_source - ' . '
2) full_path_source - PosixPath('/Users/u6533564/Documents/Office/ExecutableBookProject/example-projects/tutorials')